bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyCWLRegistry(PythonPackage):
```

Comment:
```suggestion
class PyCwlRegistry(PythonPackage):
```

Reply:
yes, or your package will be `py-c-w-l-registry` :D

Reply:
lol, yes, then I will change it, thanks for the heads up